Makefile

```diff
@@ -179,6 +179,11 @@ test-integration:
 test-odo-url-int:
 	ginkgo -v -nodes=$(TEST_EXEC_NODES) -focus="odoURLIntegration" slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -randomizeAllSpecs  tests/integration/ -timeout $(TIMEOUT)
 
+# Run push command e2e
+.PHONY: test-cmd-push
+test-cmd-push:
+	go test -v github.com/openshift/odo/tests/integration --ginkgo.focus="odo push command tests" -ginkgo.slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -ginkgo.v -timeout $(TIMEOUT)
```

Comment:
Use ```gonkgo``` instead of ```go```, so that we can enable parallel run on multiple test node. 

```
ginkgo -v -nodes=$(TEST_EXEC_NODES) -focus="odo push command tests" slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -randomizeAllSpecs  tests/integration/ -timeout $(TIMEOUT)
```